### PR TITLE
W3DScienceModelDraw

### PR DESCRIPTION
**Pull request by jonwil**

### CODE REVIEW

File: src/game/client/draw/w3dsciencemodeldraw.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ */
+#pragma once
+#include "always.h"
+#include "w3dmodeldraw.h"
+
+class W3DScienceModelDrawModuleData : public W3DModelDrawModuleData
+{
+public:
+    W3DScienceModelDrawModuleData() : m_requiredScience(SCIENCE_INVALID) {}
+    virtual ~W3DScienceModelDrawModuleData() {}
```
Comment (xezon) [inline src/game/client/draw/w3dsciencemodeldraw.h]: `override`

**Comment by xezon** [comment]
How can this be tested?

File: src/game/client/draw/w3dsciencemodeldraw.h (hunk 1 above)
Comment (jonwil) [reply src/game/client/draw/w3dsciencemodeldraw.h]: fixed.

**Comment by xezon** [comment]
W3DScienceModelDraw code is called in GLA Mission 1. No idea what it does. I played mission from start to end on both Thyme and Original and did not see any gameplay difference.

**Review by xezon — APPROVED** [review]

**Review by tomsons26 — APPROVED** [review]

**Pull request merged**